soc_026_rw0_global_gender_gap/soc_026_rw0_global_gender_gap.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul  1 14:22:50 2021
+
+@author: Jason.Winik 
+"""
+
+import pandas as pd
+from pandas import DataFrame
+import numpy as np
+from functools import reduce
+import requests
+import json
+import os
+import tabula
+utils_path = os.path.join(os.getenv('PROCESSING_DIR'),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+import logging
+from zipfile import ZipFile
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'soc_026_rw1_gender_gap_index' #check
```

Comment:
My fault for suggesting the wrong name here. Since we are updating the existing table and adding new years to a timeline, we can use the old table name. We would create a new version if the methodology changed and we were replacing the data.
```suggestion
dataset_name = 'soc_026_gender_gap_index_combined' #check
```